projects/plugins/jetpack/modules/videopress/class.jetpack-videopress.php

```diff
@@ -62,6 +62,27 @@ public function on_init() {
 
 		if ( $this->is_videopress_enabled() ) {
 			add_action( 'admin_notices', array( $this, 'media_new_page_admin_notice' ) );
+			$bridge_url = Assets::get_file_url_for_environment(
+				'modules/videopress/js/videopress-token-bridge.js',
+				'modules/videopress/js/videopress-token-bridge.js'
+			);
+
+			wp_enqueue_script(
+				'media-video-widget-extensions',
```

Comment:
Should this be renamed? Feels like it was copied from somewhere
```suggestion
				'media-video-jwt-bridge',
```